sdk/web-pubsub/web-pubsub-client/tsconfig.json

```diff
@@ -5,10 +5,11 @@
     "outDir": "./dist-esm",
     "rootDir": ".",
     "importHelpers": true,
+    "allowJs": true,
     "lib": ["DOM", "ES2017"],
     "paths": {
       "@azure/web-pubsub-client": ["./src/index"]
     }
   },
-  "include": ["src/**/*.ts", "test/**/*.ts", "samples-dev/**/*.ts"]
+  "include": ["src/**/*.ts", "src/**/*.js", "test/**/*.ts", "samples-dev/**/*.ts"]
```

Comment:
including source JS files is strange, why is `ws.browser.js` not written in TS like everything else?